packages/server/graphql/public/types/CreateOAuthAPICodePayload.ts

```diff
@@ -0,0 +1,15 @@
+import {CreateOAuthApiCodePayloadResolvers} from '../resolverTypes'
+
+export type CreateOAuthAPICodePayloadSource = {
+  code: string
+  redirectUri: string
+  state?: string | null
+}
+
+const CreateOAuthAPICodePayload: CreateOAuthApiCodePayloadResolvers = {
+  code: ({code}: CreateOAuthAPICodePayloadSource) => code,
```

Comment:
-1 put `CreateOAuthAPICodePayloadSource` in the codegen so these get added automatically. see examples in the other files in this directory